yaksh/templates/yaksh/question.html

```diff
@@ -142,7 +142,14 @@ <h3 class="brand"><strong>Online Test</h3></strong>
   <h4><u> {{ question.summary }} </u><font class=pull-right>(Marks : {{ question.points }}) </font></h4><br>
   <font size=3 face=arial> {{ question.description|safe }} </font>
   <br><font size=3 face=arial> Language: {{ question.language }} </font><br>
-
+  {% if files %}
+  <h4>Files to download for this question</h4>
+  {% for file in files %}
```

Comment:
currently student can download files by clicking on link.
we can add a button to download a zip file
I will raise an issue for this.